dask/bag/reblock.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+import numpy as np
+from dask.utils import ignoring
+def intersect_blockdims_1d(old, new):
+	summ = sum(old)
```

Comment:
Suggest glancing through [`pep8`](https://www.python.org/dev/peps/pep-0008/)

Notably a relevant style point is four space indentation.